examples/resources/incident_schedule/resource.tf

```diff
@@ -0,0 +1,76 @@
+data "incident_user" "rory" {
+  email = "rory.malcolm@incident.io"
+}
+
+data "incident_user" "martha" {
+  email = "martha@incident.io"
+}
+
+data "incident_user" "roryb" {
+  email = "rory@incident.io"
+}
+
+resource "incident_schedule" "testing_my_provider" {
+  name = "Testing Terraform"
+
+  // from tz database
+  timezone = "Europe/London"
+
+  rotations = [{
+    // A string ID for a schedule rotation
+    id                = "01HPFH8T92MPGSQS5C1SPAF4V0"
```

Comment:
How do I pick this value? Presumably we're not asking people to ULID generate it, right?

Is it not the case that most people will end up calling this something they create by hand like:
```
id = "primary"
name = "Primary"
```